integration/default_test.go

```diff
@@ -138,51 +138,5 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 
 			Eventually(container, eventuallyTimeout).Should(Serve(ContainSubstring("{\"application_status\":\"UP\"}")).OnPort(8080))
 		})
-
-		it("runs with readonly filesystem when built on tiny", func() {
```

Comment:
@pbusko What is the reasoning for removing an integration test for the readonly case? Don't you think it is a case worth testing?

Reply:
Current configuration of `tomee` relies on the fact that the `catalina-base` folder is writeable, as defined here https://github.com/paketo-buildpacks/apache-tomee/blob/main/resources/logging.properties

making `apache-tomee` to work on read-only file system requires more changes than temp dir, so there're two options:

1. keep this PR simply for fixing differences between apache-tomcat and apache-tomee + do not set temp dir in case of tiny stack.
2. convert this PR to "Make apache-tomee" run with read-only file system and include changes related to logging

Reply:
Apparently logging config wasn't the problem, tomee now starts without issues, after both `tomee.xml` and `openejb.xml` files are copied to the `$CATALINA_BASE/conf` folder

Reply:
> Apparently logging config wasn't the problem, tomee now starts without issues, after both `tomee.xml` and `openejb.xml` files are copied to the `$CATALINA_BASE/conf` folder

And it is fair to assume that when a user wants to use access logs **AND** read only root filesystem that there is a mount to `${catalina.base}/logs`, right? Maybe that is worth mentioning for `$BPL_TOMEE_ACCESS_LOGGING_ENABLED`.

Reply:
I don't think we should be writing logs to the local file system. It can cause disks to fill up unexpectedly, or write permission issues (I'm kind of surprised that's not happening here).

In Tomcat, we have a [special console handler](https://github.com/paketo-buildpacks/apache-tomcat/blob/main/resources/logging.properties#L17-L20). That is contributed by Tomee buildpack as well, but the logging.properties file isn't configured to use it. We should probably do that to ensure all of the logs are going out to the console. If that's happening some other way that's specific to Tomee, that's OK too. Then we can just remove the code that contributes this special handler.

At any rate, that's probably a different issue so I'll open one up, and we can get this merged.

Reply:
Hmm, I do see `org.apache.tomee.jul.formatter.AsyncConsoleHandler` in logging.properties, so kind of seems like we don't need the special console handler in Tomee. I'm going to move this conversation to a new issue.

Reply:
https://github.com/paketo-buildpacks/apache-tomee/issues/107